lib/components/Images/image.css

```diff
@@ -0,0 +1,33 @@
+.image-component {
+    display: flex;
+    flex-flow: column;
+    gap: 1rem;
+}
+
+.image-component iframe{
+    height: 8rem;
+}
+
+.container-img {
+    padding: 1rem;
+    box-shadow: var(--shadow-color) 0px 3px 8px;
+    border-radius: 1rem;
+}
+
+.responsive-img {
+    width: 100%;
+    height: 30rem;
```

Comment:
add object-fit property and give height 100% to images, according to the image container the images should be responsive.

Reply:
He's right, I didn't notice it. You'll need to change this